packages/valory/skills/abstract_round_abci/tests/test_tools/test_common.py

```diff
@@ -175,14 +123,14 @@ class TestBaseRandomnessBehaviourTestRunning(BaseRandomnessBehaviourTest):
 class TestBaseSelectKeeperBehaviourTestSetup(BaseCommonBaseCaseTestSetup):
     """Test BaseRandomnessBehaviourTest setup."""
 
-    _test_cls = BaseSelectKeeperBehaviourTest
+    test_cls: Type[BaseSelectKeeperBehaviourTest] = BaseSelectKeeperBehaviourTest
 
-    def run_test(self, test_instance: BaseSelectKeeperBehaviourTest) -> None:
-        """Run test"""
-        test_instance.test_select_keeper_preexisting_keeper()
+    def set_select_keeper_behaviour_class(self) -> None:
+        """Set select_keeper_behaviour_class"""
+        self.test_cls.select_keeper_behaviour_class = DummyKeeperSelectionBehaviour  # type: ignore
```

Comment:
why type ignore?

Reply:
got this on CI, tho not local
```
mypy run-test: commands[1] | mypy autonomy packages/valory plugins/aea-test-autonomy/aea_test_autonomy tests scripts --disallow-untyped-defs
packages/valory/skills/abstract_round_abci/tests/test_tools/test_common.py:67: error: Can only assign concrete classes to a variable of type "Type[BaseBehaviour]"
packages/valory/skills/abstract_round_abci/tests/test_tools/test_common.py:130: error: Can only assign concrete classes to a variable of type "Type[BaseBehaviour]"
Found 2 errors in 1 file (checked 439 source files)
```

Reply:
Makes sense. It’s a known issue with this type